src/Browser/Avalonia.Browser/webapp/modules/avalonia/generalHelpers.ts

```diff
@@ -0,0 +1,19 @@
+export class GeneralHelpers {
+    public static itemsArrayAt(instance: any, key: string): any[] {
+        const items = instance[key];
+        if (!items) {
+            return [];
+        }
+
+        const retItems = [];
+        for (let i = 0; i < items.length; i++) {
+            retItems[i] = items[i];
+        }
+        return retItems;
+    }
+
+    public static callMethod(instance: any, name: string /*, args */): any {
```

Comment:
The commented args can be removed.

Reply:
It's just shows, that we do use arguments. But it's not mapped on the actual typescript arguments list. 
It works similarly to "params" in c#, where we specify first two arguments, and everything else is read from special magic "arguments" property.